File: java/Inheritance/CargoSystemNoOOP.java
import java.util.Scanner;
public class CargoSystemNoOOP{
 	public static void main(String args[]){
	 Scanner cIn = new Scanner(System.in);
	int receiptnum, weight, charges;
	String  sname, saddress, rname, raddress; 
	
	System.out.print("Enter receipt no.: ");
	String rcpt = cIn.nextLine();
	receiptnum = Integer.parseInt(rcpt);
	
	System.out.print("Enter sender name: ");
	sname = cIn.nextLine();//cin>>sname;
	
	System.out.print("Enter sender address: ");
	 
	saddress = cIn.nextLine();//cin>>saddress;
	
	System.out.print("Enter receiver name: ");
	rname = cIn.nextLine();//cin>>rname
	
	System.out.print("Enter receiver address: ");
	raddress = cIn.nextLine();//cin>>raddress;
	
	System.out.print("Enter weight(in grams)) of parcel: ");
	weight = Integer.parseInt(cIn.nextLine());
	
	System.out.print("Enter shipment charges: ");
	charges = Integer.parseInt(cIn.nextLine());
	
	System.out.println("\n\nShipment Receipt");
	System.out.println("-------------------------");
	System.out.println("Receipt No. : "+receiptnum);
	System.out.println("Sender Name : "+sname);
	System.out.println("Sender Address : "+saddress);
	System.out.println("Receiver Name : "+rname);
	System.out.println("Receiver Address : "+raddress);
	System.out.println("Parcel Weight : "+weight+" grams");
	System.out.println("Parcel Shiping Charges : "+charges); 
 }
}

File: cpp/02 Constructor , Copy Constructor and Case Study/CPP/CPP/CargoSystem_NoOOP.cpp
#include <iostream>
#include "Parcel.cpp" 
using namespace std;


main() {
	
	int receiptnum, weight, charges;
	string sname, saddress, rname, raddress; 
	
	cout<<"Enter receipt no.: ";
	cin>>receiptnum;
	cin.ignore();
	
	cout<<"Enter sender name: ";
	getline(cin, sname);
	
	cout<<"Enter sender address: ";
	getline(cin, saddress);
	
	cout<<"Enter receiver name: ";
	getline(cin, rname);
	
	cout<<"Enter receiver address: ";
	getline(cin, raddress);
	
	cout<<"Enter weight(in grams)) of parcel: ";
	cin>>weight;
	
	cout<<"Enter shipment charges: ";
	cin>>charges; 
	cout<<"\n\nShipment Receipt"<<endl;
	cout<<"-------------------------"<<endl;
	cout<<"Receipt No. : "<<receiptnum<<endl;
	cout<<"Sender Name : "<<sname<<endl;
	cout<<"Sender Address : "<<saddress<<endl;
	cout<<"Receiver Name : "<<rname<<endl;
	cout<<"Receiver Address : "<<raddress<<endl;
	cout<<"Parcel Weight : "<<weight<<" grams"<<endl;
	cout<<"Parcel Shiping Charges : "<<charges<<endl;
	  
	system("pause");
}

